conda-recipe/run_test.sh

```diff
@@ -0,0 +1 @@
+nosetests -v metadatastore --cover-package metadatastore --with-coverage
```

Comment:
Does this get run on every build and does this require a running mongo instance?